Source/WebCore/bindings/js/JSWorkerGlobalScopeBase.cpp

```diff
@@ -156,6 +159,25 @@ String JSWorkerGlobalScopeBase::codeForEval(JSC::JSGlobalObject* globalObject, J
     return nullString();
 }
 
+bool JSWorkerGlobalScopeBase::canCompileStrings(JSC::JSGlobalObject* globalObject, JSC::CompilationType compilationType, String codeString, JSC::JSValue bodyArgument)
```

Comment:
Really annoying that this code is basically exactly the same as `JSDOMWindowBase::canCompileStrings`. Can we share more code?

Reply:
I'll have a look to see if I can move it to a shared function somewhere